### PR TITLE
WT-8921 Add a test to verify RTS won't update "logged" tables under in-memory configuration

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

File: test/suite/test_rollback_to_stable33.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    scenarios = make_scenarios(format_values, logged)
+
+    # Configure an in-memory database.
+    conn_config = 'in_memory=true'
+
+    # Smoke test RTS on in-memory databases.
+    def test_rollback_to_stable33(self):
+        uri = "table:rollback_to_stable33"
+        ds_config = ',log=(enabled=true)' if self.logged else ',log=(enabled=false)'
+        self.session.breakpoint()
```
Comment (quchenhao) [inline test/suite/test_rollback_to_stable33.py:53]: Do you need this anymore?

File: test/suite/test_rollback_to_stable33.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        ds.populate()
+
+        # Make changes at timestamp 30.
+        c = self.session.open_cursor(uri, None, None)
+        self.session.begin_transaction()
+        c[ds.key(10)] = ds.value(100)
+        c[ds.key(11)] = ds.value(101)
+        c[ds.key(12)] = ds.value(102)
+        self.session.commit_transaction('commit_timestamp=30');
+        c.close
```
Comment (quchenhao) [inline test/suite/test_rollback_to_stable33.py:65]: c.close()?

File: test/suite/test_rollback_to_stable33.py (hunk 1 above)
Comment (keithbostic) [reply test/suite/test_rollback_to_stable33.py]: Fixed in cf04af0.

File: test/suite/test_rollback_to_stable33.py (hunk 2 above)
Comment (keithbostic) [reply test/suite/test_rollback_to_stable33.py]: Fixed in b6bbd07.

**Review by quchenhao — APPROVED** [review]
lgtm

**Pull request merged**